crates/solidity/testing/perf/npm/src/@types/solc06/index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module "solc06" {
+  class Compiler {
```

Comment:
I made up this name, I only know that the module has a default object that contains the compile function. However, this trick isn't working: 

```
crates/solidity/testing/perf/npm/src/solc.runner.mts(55,50): error TS2339: Property 'compile' does not exist on type 'typeof Compiler'.
```

Reply:
Is the default export expected to be a class, or an object/instance of that class?
Looking at the README of https://www.npmjs.com/package/solc, it looks like a function instead? i.e.:

```ts
export default {
  compile: (input: string, options: CompilationOptions) => string;
}
```

Reply:
On a parallel note:
Rather than adding multiple dependencies here, and defining each of their types separately, I wonder if `solc.loadRemoteVersion()` would be easier to use here, to load them up during the `fetch` stage, so that we get a single API to call?
This also means we can dynamically choose the list of versions to load dynamically based on the test dataset, rather than maintaining both lists manually.

Reply:
ah good point, I missed that function. we can't use it at fetch, because it doesn't save the state (and it doesn't cache it).